bugsnag-plugin-android-ndk/src/main/jni/utils/serializer/event_reader.c

```diff
@@ -205,41 +250,50 @@ bugsnag_event *bsg_report_v8_read(int fd) {
 
   // read the feature flags, if possible
   bsg_read_feature_flags(fd, &event->feature_flags, &event->feature_flag_count);
+  // TODO: read the opaque data, if possible
```

Comment:
Is this TODO supposed to be here?

Reply:
Yes - it'll be completed in another ticket